numpy/random/tests/test_random.py

```diff
@@ -352,40 +356,41 @@ def test_bytes(self):
         np.testing.assert_equal(actual, desired)
 
     def test_shuffle(self):
```

Comment:
I'm not sure what you mean, but I think you'll find that non-generator tests will drop you into the errored test, and you can `u` you way to the enclosing test function to inspect the stack, but you can't do that with the `yield` test functions.